src/Raven.Server/Program.cs

```diff
@@ -35,7 +36,19 @@ namespace Raven.Server
     public class Program
     {
         private static readonly Logger Logger = LoggingSource.Instance.GetLogger<Program>("Server");
-        
+
+        static Program()
+        {
+            bool useLegacyHttpClientFactory = false;
+            var useLegacyHttpClientFactoryAsString = Environment.GetEnvironmentVariable("RAVEN_HTTP_USELEGACYHTTPCLIENTFACTORY");
+            if (useLegacyHttpClientFactoryAsString != null && bool.TryParse(useLegacyHttpClientFactoryAsString, out useLegacyHttpClientFactory) == false)
+                throw new InvalidOperationException($"Could not parse 'RAVEN_HTTP_USELEGACYHTTPCLIENTFACTORY' env variable with value '{useLegacyHttpClientFactoryAsString}'.");
+
+            RequestExecutor.HttpClientFactory = useLegacyHttpClientFactory 
```

Comment:
can we change this on the fly? should we?

Reply:
No, it is internal at the moment